alibi/api/interfaces.py

```diff
@@ -22,6 +22,7 @@ def default_meta() -> dict:
         "type": [],
         "explanations": [],
         "params": {},
+        "version": None,
```

Comment:
This is not strictly necessary here but added it as it serves as a kind of high-level "schema".